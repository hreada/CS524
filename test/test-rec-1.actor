rec(lambda(y, lambda(x, seq(send(x,5),become(y)))))
