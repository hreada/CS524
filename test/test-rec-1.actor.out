<T>
  <actors>
    <actor>
      <k> exec ( lambda ( x , seq ( send ( x , 5 ) , become ( lambda ( _2 , app ( app ( lambda ( _1 , app ( lambda ( y , lambda ( x , seq ( send ( x , 5 ) , become ( y ) ) ) ) , lambda ( _2 , app ( app ( _1 , _1 ) , _2 ) ) ) ) , lambda ( _1 , app ( lambda ( y , lambda ( x , seq ( send ( x , 5 ) , become ( y ) ) ) ) , lambda ( _2 , app ( app ( _1 , _1 ) , _2 ) ) ) ) ) , _2 ) ) ) ) ) ) </k>
      <id> initactor </id>
    </actor>
  </actors>
  <messages> .MessageCellBag </messages>
  <definedaddr> ListItem ( initactor ) </definedaddr>
</T>
